src/debugAdapter/goDebug.ts

```diff
@@ -1342,6 +1363,42 @@ class GoDebugSession extends LoggingDebugSession {
 				Scope: scope,
 				Cfg: this.delve.loadConfig
 			};
+
+		// If coming from watch/variables pane via the 'Copy Value' context menu action,
+		// first fetch the return value of the expression and override the loadConfig
+		// to get the complete value in a second, final request
+		let evalSymbolsConfig = Object.assign({}, this.delve.loadConfig);
+		let fetchReturnValue: EvalOut = null;
+		if (!this.delve.isApiV1 && (args.context === 'variables')) {
```

Comment:
I would refactor this to make a second call only if the string length is greater than maxStringLen.